PhysicsTools/TagAndProbe/test/Electron_TagProbeTreeProducer_cfg.py

```diff
@@ -389,7 +389,6 @@
 ##   |_____/_/\_\\__\___|_|  |_| |_|\__,_|_|    \_/ \__,_|_|  |___/
 ##   
 ## Here we show how to use a module to compute an external variable
-## process.load("JetMETCorrections.Configuration.DefaultJEC_cff")
 ## ak5PFResidual.useCondDB = False
```

Comment:
This modifies a module from the line removed, can also get removed

Reply:
Done. Thanks.